block/internal/da/local_blob_api.go

```diff
@@ -0,0 +1,80 @@
+package da
```

Comment:
nit: can this file be marked as _tests? or a comment at the top saying its for tests